src/client_impl.h

```diff
@@ -59,6 +60,18 @@ class MixerClientImpl : public MixerClient {
   std::string deduplication_id_base_;
   std::atomic<std::uint64_t> deduplication_id_;
 
+  // Atomic objects for recording statistics.
+  // check cache hit rate:
+  // 1 - total_blocking_remote_check_calls_ / total_check_calls_.
```

Comment:
Your formula is for cache miss rate, not hit rate

Reply:
I have changed it to calculating cache miss rate.